permissions/src/main/java/gov/nasa/jpl/aerie/permissions/exceptions/Unauthorized.java

```diff
@@ -0,0 +1,25 @@
+package gov.nasa.jpl.aerie.permissions.exceptions;
+
+import gov.nasa.jpl.aerie.permissions.Action;
+import gov.nasa.jpl.aerie.permissions.PermissionType;
+import gov.nasa.jpl.aerie.permissions.gql.PlanId;
+
+public class Unauthorized extends Exception {
+  public Unauthorized(final String role, final Action action) {
+    super("Role '" + role + "' is not allowed to perform action '" + action + "'");
```

Comment:
Very minor, but we could replace all the strings in these super calls with text blocks and `formatted` as a standard.

```suggestion
    super("""Role "%s" is not allowed to perform action "%s".""".formatted(role, action));
```

Reply:
Done!